backend/app/infrastructure/routes/rewards.py

```diff
@@ -413,3 +428,21 @@ def get(self, epoch: int):
         app.logger.debug(f"Merkle tree leaves for epoch {epoch}: {merkle_tree_leaves}")
 
         return merkle_tree_leaves.to_dict()
+
+
+@ns.route("/current_user_budget")
```

Comment:
let's try to keep the API more RESTful - `/budget/<string:user_address>/estimated`

Reply:
Changed to /budget/<string:user_address>/upcoming since we already use estimated for the calculation simulations before allocating GLMs

Reply:
ok, but we could also follow the convention of other endpoints that we pass `address` in the url, not as a req body